### PR TITLE
{vis}[GCCcore/11.3.0] gifsicle v1.93

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
himem01.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/c0c74f2f04e7b06a9913431e995fed57 for a full test report.

File: easybuild/easyconfigs/g/gifsicle/gifsicle-1.93-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+checksums = ['941c0b1c3ebc748949a0d14363f0fb3c9d2c06ee122b3548315b9233eb956d88']
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('Autotools', '20220317'),
+]
+
+configure_cmd = "autoreconf -i && ./configure"
```
Comment (jfgrimm) [inline easybuild/easyconfigs/g/gifsicle/gifsicle-1.93-GCCcore-11.3.0.eb:26]:
```suggestion
preconfigopts = "autoreconf -i &&"
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]
you're missing some (runtime) dependencies:

```
ldd $EBROOTGIFSICLE/bin/gifview 
	linux-vdso.so.1 =>  (0x00007fffb25ed000)
	libSM.so.6 => /lib64/libSM.so.6 (0x00007febe5ef8000)
	libICE.so.6 => /lib64/libICE.so.6 (0x00007febe5cdc000)
	libX11.so.6 => /lib64/libX11.so.6 (0x00007febe599e000)
	libm.so.6 => /lib64/libm.so.6 (0x00007febe569c000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007febe5480000)
	libc.so.6 => /lib64/libc.so.6 (0x00007febe50b2000)
	libuuid.so.1 => /lib64/libuuid.so.1 (0x00007febe4ead000)
	libxcb.so.1 => /lib64/libxcb.so.1 (0x00007febe4c85000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007febe4a81000)
	/lib64/ld-linux-x86-64.so.2 (0x00007febe6100000)
	libXau.so.6 => /lib64/libXau.so.6 (0x00007febe487d000)
```

File: easybuild/easyconfigs/g/gifsicle/gifsicle-1.93-GCCcore-11.3.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/g/gifsicle/gifsicle-1.93-GCCcore-11.3.0.eb:26]: Thanks, that is a better way.

**Comment by sassy-crick** [comment]
> you're missing some (runtime) dependencies:
> 
> ```
> ldd $EBROOTGIFSICLE/bin/gifview 
> 	linux-vdso.so.1 =>  (0x00007fffb25ed000)
> 	libSM.so.6 => /lib64/libSM.so.6 (0x00007febe5ef8000)
> 	libICE.so.6 => /lib64/libICE.so.6 (0x00007febe5cdc000)
> 	libX11.so.6 => /lib64/libX11.so.6 (0x00007febe599e000)
> 	libm.so.6 => /lib64/libm.so.6 (0x00007febe569c000)
> 	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007febe5480000)
> 	libc.so.6 => /lib64/libc.so.6 (0x00007febe50b2000)
> 	libuuid.so.1 => /lib64/libuuid.so.1 (0x00007febe4ead000)
> 	libxcb.so.1 => /lib64/libxcb.so.1 (0x00007febe4c85000)
> 	libdl.so.2 => /lib64/libdl.so.2 (0x00007febe4a81000)
> 	/lib64/ld-linux-x86-64.so.2 (0x00007febe6100000)
> 	libXau.so.6 => /lib64/libXau.so.6 (0x00007febe487d000)
> ```

Thanks for finding that. I have added the missing X11 so hopefully that is ok now.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.5, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/df3ccb3e2bc69e10796667a7eb1820af for a full test report.

**Comment by Micket** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**